front-end/src/components/Post/GovernanceSideBar/Referenda/ReferendumVoteInfo.tsx

```diff
@@ -62,54 +120,53 @@ const ReferendumVoteInfo = ({ className, referendumId, threshold }: Props) => {
 		let unsubscribe: () => void;
 
 		api.query.balances.totalIssuance((result) => {
-			setElectorate(result);
+			setTotalIssuance(result);
 		})
 			.then( unsub => {unsubscribe = unsub;})
 			.catch(console.error);
 
 		return () => unsubscribe && unsubscribe();
 	},[api, isApiReady]);
 
-	useEffect(() => {
-		if (!threshold){
-			console.error('No threshold set');
-			return;
-		}
-
-		const res = getPassingThreshold(nayVotes, electorate, threshold);
-		if (res.isValid && res.passingThreshold) {
-			setPassingThreshold(res.passingThreshold);
-		}
-	}, [electorate, nayVotes, threshold, turnout]);
-
 	return (
-		<Card className={loadingStatus.isLoading ? `LoaderWrapper ${className}` : className}>
-			{loadingStatus.isLoading
-				?
-				<Loader text={loadingStatus.message} timeout={30000} timeoutText='Api is unresponsive.'/>
-				:
-				<>
-					<VoteProgress
-						ayeVotes={ayeVotes}
-						className='vote-progress'
-						passingThreshold={passingThreshold}
-						nayVotes={nayVotes}
-					/>
-					<Grid columns={2} divided>
-						<Grid.Row>
-							<Grid.Column>
-								<h6>Turnout</h6>
-								<div>{formatBnBalance(turnout, { numberAfterComma: 2 })}</div>
-							</Grid.Column>
-							<Grid.Column>
-								<h6>Electorate</h6>
-								<div>{formatBnBalance(electorate, { numberAfterComma: 2 })}</div>
-							</Grid.Column>
-						</Grid.Row>
-					</Grid>
-				</>
-			}
-		</Card>
+		<>
+			<PassingInfo isPassing={isPassing}/>
+			<Card className={loadingStatus.isLoading ? `LoaderWrapper ${className}` : className}>
+				{loadingStatus.isLoading
+					? <Loader text={loadingStatus.message} timeout={30000} timeoutText='Api is unresponsive.'/>
+					: <>
+						{
+							isPassing === null
+								? <Loader className={'progressLoader'} text={'Loading vote progress'} timeout={30000} timeoutText='Vote calculation failed'/>
+								: <VoteProgress
+									ayeVotes={ayeVotes}
+									className='vote-progress'
+									isPassing={isPassing}
+									threshold={getThreshold}
+									nayVotes={nayVotes}
+								/>
+						}
+
+						<Grid columns={3} divided>
+							<Grid.Row>
+								<Grid.Column>
+									<h6>Turnout {turnoutPercentage > 0 && <span className='turnoutPercentage'>({turnoutPercentage}%)</span>}</h6>
+									<div>{formatBnBalance(turnout, { numberAfterComma: 2 })}</div>
```

Comment:
Wonder if to add the token symbol here and to all votes without conviction (lines 158 and 162).

Reply:
Sure why now. I'm wondering the amount of text on this screen is getting high. What about actually removing the "KSM" from the progressbar? After all users know we talk about KSM don't you think?

Reply:
Yeah, adding here and removing from progress bar sound good

Reply:
Tested and agree with you. Too much text. Let's also remove it from the progress bar